rest-api-spec/src/main/resources/rest-api-spec/test/get/70_source_filtering.yml

```diff
@@ -66,3 +66,37 @@
   - match:   { _id:      "1"       }
   - match:   { fields.count:  [1] }
   - match:   { _source.include.field1: v1 }
+
+---
+"Deprecated _source_include and _source_exclude":
+
+  - skip:
+      version: " - 6.4.99"
```

Comment:
// TODO consider changing the version (if needed) once the clients specs are changed to `_source_includes` & `_source_excludes`

Reply:
I'd use 6.99.99 for now. After I merge this I'll backport it to the 6.x branch. Then I'll come in and update the range here to include the backport target version.

I'm *fairly* sure this is what caused the build to fail. If you crack open the PR build's full console it'll have a `REPRODUCE WITH` line that should reproduce the failure. And I'm fairly sure switching this to 6.99.99 will make it go away.